### DISCUSSION TITLE
Slice Thickness issue - cannot increase it higher then 1.5mm in Siemens Scanner for a MSE seq file

### DISCUSSION THREAD

**Discussion by ZemaTimoteo** [pulseq/pulseq#65] [Q&A]
Dear all

I created a MSE sequence with vFA, but I am being unable to run this code in the SIEMENS scanner when I run it for higher slice thickness than 1.5mm. 
I am able to generate a '.seq' file, but I when I run it in the SIEMENS scanner, it fails to compile and stop the scan.
The only parameter i change is the slice thickness from 1.5mm to 3mm. 
Does anyone have an idea what could be the issue? I checked, slew rate, amplitude, SAR, and all checks-up.

Any help would be mostly appreciated.

I can give more details if needed

Best
Tiago

**Comment by m-a-x-i-m-z** [comment]
Hi!

did you try to simulate your sequence in the IDEA environment? It may be helpful to see the error message.
